### PR TITLE
Monorepo ci test

### PR DESCRIPTION
**Pull request by mandresm**
Is this ready to be merged?

### CODE REVIEW

**Comment by pgierz** [comment]
Not yet, I still need to actually add the tests. I'll do that after I'm back from my Covid check. 

The large number of changes seems like my vim accidentally ran black formatting. Would you agree with that assessment, or should I look through it more carefully?

File: src/esm_runscripts/namelists.py [hunk 1]
```diff
@@ -265,8 +267,8 @@ def echam_transient_forcing(config):
                 import pandas as pd
 
                 try:
-                    forcing_table = pd.read_csv(
-                        config["echam"]["transient_forcing_table"]
+                    forcing_table = pd.open_csv(
+                        config["echam"]["transient_forcing_table"], sep=";", index_col=0
```
Comment (mandresm) [inline src/esm_runscripts/namelists.py]: My only comment is here: `open_csv` didn't seem to exist and we changed to `read_csv` so, why is `open_csv` back? Is this once again one of this times where GitHub is performing poorly on showing the changes of a merge?

For the rest of the changes, I did not spot anything that affects to any other ESM-Tools functionalities.

Comment (pgierz) [reply src/esm_runscripts/namelists.py]: That's GitHub being messy again...

Comment (pgierz) [reply src/esm_runscripts/namelists.py]: Check again please, I just fixed it...

**Comment by pgierz** [comment]
I finally found the error I was having with generalized tests (spaces aren't allowed in some parts of GitHub Actions, because. well.. why make it easy 🤦) I'll push some FESOM CI stuff after my break, and we can then expand that to other models easily enough

File: src/esm_runscripts/namelists.py (hunk 1 above)
Comment (mandresm) [reply src/esm_runscripts/namelists.py]: Looks good to me, from my side, feel free to merge whenever you consider it's ready.

**Comment by pgierz** [comment]
Tests are in place for:

* Echam standalone
* Fesom Standalone
* Pism Standalone
* AWIESM 
* AWICM

We should probably still discuss how to tweak them, right now they take a long time and are triggered rather easily. I will look into how to have multiple runners active on Ollie, that should somehow be possible, but I need to discuss with Natalja and Malte first.

**Comment by pgierz** [comment]
@mandresm this is ready as far as I can tell. There are still some things we could tweak and optimize (mostly on my side for the ollie CI) but if no one complains by tomorrow morning I'll put this one in too. One less PR :-)

**Pull request merged**